model_compression_toolkit/gptq/common/gptq_config.py

```diff
@@ -78,7 +77,7 @@ def __init__(self,
             optimizer_bias (Any): Optimizer to override the rest optimizer for bias.
             log_norm (bool): Whether to use log normalization to the GPTQ Jacobian-based weights.
             weights_n_iter (int): Number of random iterations to run Jacobian approximation for GPTQ weights.
-            quantizer_config (GPTQQuantizerConfig): A class that contains the quantizer specific config.
+            entropy_reg (float): A floating point number that defines the gumbel regularization factor.
```

Comment:
maybe change to regularization_factor. relevant for all quantizers, not only gumbel